### PR TITLE
Fix for bug 92988: disable sql_require_primary_key while [re]creating general_log/slow_log

### PR DESCRIPTION
**Pull request by sjmudd**
See: [bug 92988](https://bugs.mysql.com/bug.php?id=92988).

### CODE REVIEW

File: scripts/mysql_system_tables.sql [hunk 1]
```diff
@@ -205,12 +205,18 @@ CREATE TABLE IF NOT EXISTS time_zone_leap_second (   Transition_time bigint sign
 
 CREATE TABLE IF NOT EXISTS procs_priv ( Host char(60) binary DEFAULT '' NOT NULL, Db char(64) binary DEFAULT '' NOT NULL, User char(32) binary DEFAULT '' NOT NULL, Routine_name char(64) COLLATE utf8_general_ci DEFAULT '' NOT NULL, Routine_type enum('FUNCTION','PROCEDURE') NOT NULL, Grantor char(93) DEFAULT '' NOT NULL, Proc_priv set('Execute','Alter Routine','Grant') COLLATE utf8_general_ci DEFAULT '' NOT NULL, Timestamp timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP, PRIMARY KEY (Host,Db,User,Routine_name,Routine_type), KEY Grantor (Grantor) ) engine=InnoDB STATS_PERSISTENT=0 CHARACTER SET utf8 COLLATE utf8_bin   comment='Procedure privileges' TABLESPACE=mysql;
 
+-- bug#92988: Temporarily turn off sql_require_primary_key for the
+-- next 2 tables as this is not necessary as they do not replicate.
```
Comment (dveeden) [inline scripts/mysql_system_tables.sql:209]: PK also improve performance for non-replicated InnoDB, but this isn't InnoDB.

Comment (sjmudd) [reply scripts/mysql_system_tables.sql:209]: Indeed.

**Comment by sjmudd** [comment]
I confirm the code being submitted is offered under the terms of the OCA, and that I am authorized to contribute it.

**Comment by sjmudd** [comment]
The bot is rather lazy, taking 19 hours to ask me. Does it sleep too much?

**Comment by mysql-admin** [comment]
@sjmudd: There is a manual verification step so the bot is set to run once in 24 hours (unless triggered by the manual verification step)

Thanks for your contribution 
==Omer

**Pull request closed without merging**